libs/dyn/value.go

```diff
@@ -154,7 +154,19 @@ func (v Value) eq(w Value) bool {
 		// This is safe because we don't allow slices to be mutated.
```

Comment:
I think you can remove these comments

Reply:
Good catch, I moved this below to where the ptr comparison happens.